runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/PTransformMatchers.java

```diff
@@ -168,6 +171,21 @@ public boolean matches(AppliedPTransform<?, ?, ?> application) {
     };
   }
 
+  public static PTransformMatcher createViewWithViewFn(final Class<? extends ViewFn> viewFnType) {
```

Comment:
The PR is already merged, but could be valuable to add a javadoc for this method.